server/game/Effects/StaticEffect.js

```diff
@@ -45,7 +59,7 @@ const conflictingEffects = {
 };
 
 class StaticEffect {
-    constructor(type = '', value) {
```

Comment:
why did you change this?

Reply:
Actually I think that was you as part of the Festival of the Departed PR 😛